tests/components/cloud/test_ai_task.py

```diff
@@ -46,6 +48,21 @@ def mock_cloud_ai_task_entity(hass: HomeAssistant) -> CloudLLMTaskEntity:
     return entity
 
 
+async def test_setup_entry_skips_when_not_logged_in(
+    hass: HomeAssistant,
+) -> None:
+    """Test setup_entry exits early when not logged in."""
+    cloud = MagicMock()
+    cloud.is_logged_in = False
+    entry = MockConfigEntry(domain="cloud")
+    entry.add_to_hass(hass)
+    hass.data[DATA_CLOUD] = cloud
+
+    async_add_entities = AsyncMock()
+    await async_setup_entry(hass, entry, async_add_entities)
```

Comment:
We don't test platform details directly like this.

Set up the config entry and assert (missing) state in the core state machine.